README.md

```diff
@@ -5,7 +5,7 @@
 * GLFW C library source is included and built automatically as part of the Go package. But you need to make sure you have dependencies of GLFW:
 	* On macOS, you need Xcode or Command Line Tools for Xcode (`xcode-select --install`) for required headers and libraries.
 	* On Ubuntu/Debian-like Linux distributions, you need `libgl1-mesa-dev` and `xorg-dev` packages.
-	* On CentOS/Fedora-like Linux distributions, you need `libX11-devel libXcursor-devel libXrandr-devel libXinerama-devel mesa-libGL-devel libXi-devel libXxf86vm-devel` packages.
+	* On CentOS/Fedora-like Linux distributions, you need `mesa-libGL-devel`. To build for X11, you'll need `libX11-devel libXcursor-devel libXrandr-devel libXinerama-devel libXi-devel libXxf86vm-devel` packages; and for Wayland you'll need `libxkbcommon-devel wayland-devel`; if you're using Gnome with Wayland, you should also install `libdecor-devel`.
```

Comment:
I don't think you should explicitly call out Gnome here. Libdecor solves decorations for more systems and it's not really about the developer's experience but the user running the releases application. Maybe just write that it is an optional and recommended dependency for improved window docorarions on Wayland?